keanu-project/src/main/java/io/improbable/keanu/util/Py4jUtils.java

```diff
@@ -0,0 +1,43 @@
+package io.improbable.keanu.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+@UtilityClass
+public class Py4jUtils {
+
+    public byte[] toByteArray(double[] doubleArray) {
+        ByteBuffer doubleBuffer = ByteBuffer.allocate(Double.SIZE / Byte.SIZE * doubleArray.length);
+        // Java defaults to BIG_ENDIAN. LITTLE_ENDIAN is what is expected by numpy to construct an ndarray.
+        doubleBuffer.order(ByteOrder.LITTLE_ENDIAN);
+        for (double element : doubleArray) {
+            doubleBuffer.putDouble(element);
+        }
+        byte[] byteArray = doubleBuffer.array();
+        return byteArray;
+    }
+
+    public byte[] toByteArray(int[] integerArray) {
+        ByteBuffer integerBuffer = ByteBuffer.allocate(Integer.SIZE / Byte.SIZE * integerArray.length);
+        // Java defaults to BIG_ENDIAN. LITTLE_ENDIAN is what is expected by numpy to construct an ndarray.
+        integerBuffer.order(ByteOrder.LITTLE_ENDIAN);
+        for (int element : integerArray) {
+            integerBuffer.putInt(element);
+        }
+        byte[] byteArray = integerBuffer.array();
+        return byteArray;
+    }
+
+    public byte[] toByteArray(Boolean[] booleanArray) {
```

Comment:
nit - for consistency with the other 2 methods

```suggestion
    public byte[] toByteArray(boolean[] booleanArray) {
```